templates/admin/config.tmpl

```diff
@@ -237,7 +237,7 @@
 						{{.CsrfTokenHtml}}
 						<div class="inline field ui left">
 							<div class="ui input">
-								<input type="email" name="email" required>
+								<input type="email" name="email" placeholder="{{.i18n.Tr "admin.config.test_email_placeholder"}}" required>
```

Comment:
```suggestion
								<input type="email" name="email" placeholder="{{.i18n.Tr "admin.config.test_email_placeholder"}}" size="29" required>
```

So that the text is not cut off:

<img width="588" alt="image" src="https://user-images.githubusercontent.com/115237/89708283-f69b5f00-d975-11ea-94b5-40d1f68efa64.png">